test-e2e/app/map/map.page.e2e.ts

```diff
@@ -0,0 +1,198 @@
+/**
+ * Created by vs on 13/4/16.
+ */
+describe('Map Page ', function() {
+  jasmine.DEFAULT_TIMEOUT_INTERVAL = 100000;
+
+  var countryLink = new Array();
+  countryLink = element.all(by.css('.country-name'));
+  var mapImg = element(by.css('.map-color'));
+  var iconHomes = element(by.css('.home>img'));
+  var messageAboutMap = 'Map is not loaded';
+  var messageAboutCountryLink = 'Page is not loaded';
+  var searchButton = element(by.css('.matrix-search-button>img'));
+  var searchField = element(by.css('#search'));
+  var itemsAroundTheWorld = element.all(by.css('.search-item-name')).last();
+  var countryLetter = element.all(by.css('.country-letter'));
+  var country = element.all(by.css('span[class*="country-name"]'));
+  var littleHeaderIcon = element(by.css('.search-title-thing-icon.pull-left.icon'));
+
+  var EC = protractor.ExpectedConditions;
+  var TIMEOUT = 10000;
+  browser.manage().window().maximize();
+  browser.manage().timeouts().pageLoadTimeout(75000);
+  /*
+   Open Map Link
+   Click on every Link with Country
+   Write the Text to Colsole about checked link
+   */
+
+  /*   beforeEach(function(done){ //TODO This code need for starting tests on Mozilla Firefox. Start
+   browser.get('/map');
+   browser.waitForAngular();
+   $('body').isPresent().then(function() {
+   done();
+   }, function (){
+   done();
+   });
+   browser.wait(EC.visibilityOf(mapImg), TIMEOUT, messageAboutMap);
+   }); //TODO This code need for starting tests on Mozilla Firefox. Finish*/
+
+  beforeAll(function(){ //TODO This code need for starting tests on Chrome. Start
+    browser.get('/map');
+    browser.wait(EC.visibilityOf(mapImg), TIMEOUT, messageAboutMap);
+  }); //TODO This code need for starting tests on Chrome. Finish
+
+  it ('Check country Bangladesh', function () {
+    browser.sleep(2000);
+    var elem = country.first();
+    expect(elem.getText()).toEqual('Bangladesh');
+  });
+
+  it ('Check country Colombia', function () {
+    var elem = country.get(7);
```

Comment:
this kind of `get` will be dead as soon as content changes
but it should NOT
use smart selection (by contry id or what ever, something more persistant)